tests/swagger_generation/test_swagger_generator.py

```diff
@@ -234,7 +243,8 @@ class Error(m.Schema):
             produces=produces,
             title=title,
             version=version,
-            default_response_schema=Error()
+            default_response_schema=Error(),
+            tags=[Tag(name='bar', external_docs=ExternalDocumentation(url='http://bardocs.com'))]
```

Comment:
should we also put in a description here just for completeness?

Reply:
Ah, sorry I was being lazy.
I'll add this.